src/SFA.DAS.Admin.Roatp.Web/Models/SelectTrainingProviderSubmitViewModel.cs

```diff
@@ -2,8 +2,6 @@
 
 public class SelectTrainingProviderViewModel : SelectTrainingProviderSubmitViewModel
 {
-    // this is a temporary field only needed to allow testing.  Will be removed in CSP-2210
-    public string MatchedResult { get; set; } = string.Empty;
 }
 
 public class SelectTrainingProviderSubmitViewModel : IBackLink
```

Comment:
IBackLink should be on the ViewModel, also your sonar is complaining about your view model being empty.

Reply:
I've made it a single viewModel, which addresses these issues